src/communication.rs

```diff
@@ -54,6 +55,13 @@ pub struct Sender {
     connection_id: u32,
 }
 
+impl fmt::Debug for Sender {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "Sender {{ token: {:?}, channel: _, connection_id: {:?} }}",
```

Comment:
I'd write the type here instead of _

Reply:
Is there a way to print the type parameter to `mio::channel::SyncSender`? Doesn't seem like it, so would just
```
Sender { token: ..., channel: mio::channel::SyncSender, connection_id: ... }
```
be enough?

Reply:
Oh, I was forgetting that we know the type parameter is `Command`.